examples/prometheus/prometheus.yaml

```diff
@@ -117,7 +124,8 @@ objects:
           - -tls-key=/etc/tls/private/tls.key
           - -client-secret-file=/var/run/secrets/kubernetes.io/serviceaccount/token
           - -cookie-secret-file=/etc/proxy/secrets/session_secret
-          - -skip-auth-regex=^/metrics
+          - -openshift-delegate
+          - '-openshift-resources={"/": {"resource": "namespaces", "verb": "get", "name": "${NAMESPACE}"}}'
```

Comment:
Can you test with the :v1 tag?  If that works, I'll promote it to :latest and then merge this at the same time.